tests/functional/modules/test_zos_fetch_func.py

```diff
@@ -182,12 +192,12 @@ def test_fetch_uss_file_replace_on_local_machine(ansible_zos_module):
 
 def test_fetch_uss_file_present_on_local_machine(ansible_zos_module):
     hosts = ansible_zos_module
+    dest_path = get_unique_uss_file_name()
     params = {
         "src":"/etc/profile",
-        "dest":"/tmp/",
+        "dest": dest_path,
```

Comment:
With this new change the test is no longer doing what it is supposed to, name is `test_fetch_uss_file_present_on_local_machine`, but the file that copies into is not present.